less/index.less

```diff
@@ -52,8 +52,139 @@ table {
 * {
     box-sizing: border-box;
 }
+body {
+	height: 100%;
+	// border: 1px solid black;
+  }
+@main-bg:        white;
+@font-stack: 'Roboto', Arial, sans-serif;
+.container{
+	// height: 100%;
+	width: 1024px;
+  	margin: 0 auto;
+  	background: @main-bg;
+  	font-family: @font-stack;	
+}
+.custom-flex( @justify, @align ){
+	display: flex;
+	justify-content: @justify;
+	align-item:@align;
+  }
+  header nav{
+	.custom-flex(flex-end, space-between);
+	padding: 0%;
+	a{text-decoration: none;}
+	
+	 a:hover {
+		text-decoration: underline;
+	  }
+  }
+  header nav a{
+	  padding: 0% 2%;
+	  margin: 1% 0% 0 0;
+  }
+  
+  .top-section img{
+	  margin:3% 0% 1% 3%;
+  }
+  .hfont(){
+	  font-size: 200%;
+  }
+  .pfont(){
+	  font-size: 100%;
+  }
+  .inmarleft(){
+	  margin-left:3%;
+  }
+  .top-section{
+	background: pink;
+  // .custom-flex(inline-block, center)
+	  display: inline-block;
+	  line-height: 150%;
+	  margin: 2% 0 2% 0%;
+	h1{
+	  .hfont;
+	  .inmarleft;
+	  padding-bottom: 1%;
+	 }
+	 p{
+		 .pfont;
+		 .inmarleft;
+		 padding-bottom: 2%;
 
-html, body {
-    height: 100%;
+	 }
+	 button{
+		 padding: .5%;
+		 border-radius: 10%;
+		 border: 1px solid navy;
+		 cursor: pointer;
+		 background:grey;
+		 margin: 0% 0% 5% 3%;
+		 color: white;
+
+	 }
+  }
+  .custom-table(){
+	font-family: arial, sans-serif;
+	border-collapse: collapse;
+	width: 100%;
 }
+  .mid-section{
+	  background: #FFFBFB;
+	  display: flex;
+	  line-height: 150%;
+	  margin: 2% 0 2% 0%;
+		// .a{ padding-right: 1%;
+		// }
+		// .b{ padding-left: 1%;}
+		table{
+			.custom-table;
+			border: 1px solid black;
+			
+		tr{	
+			text-align: left;
+
+		}
+	}
+	  .mid-container{
+		width: 100%;
+		h1{
+			.hfont;
+			margin: 5% 0;
+		}
+		p{
+			.pfont;
+		}
+		display : inline-block;
+		
+	  }
+  }
+  .last-section{
+	display: block;
+	h1{
+		.hfont;
+		padding:2% 0;
+	}
+
+	table{
+		.custom-table;
+		padding: 2% 0;
+		th, td {
+			border-top: 1px solid #dddddd;
```

Comment:
This color is repeated a few times. It would have been good to use a mixin for it.